### DISCUSSION TITLE
maxExpansions - the maximum number of terms to match

### DISCUSSION THREAD

**Discussion by anshulgoel27** [BurntSushi/fst#173] [Q&A] [answered]
I am trying to use this crate for fuzzy searching, but I am not able to find anything related to use of maxExpansions - the maximum number of terms to match in fuzzy search.

Can I do it using this crate somehow?

**Comment by BurntSushi** [comment, marked as answer]
This crate doesn't really provide fuzzy search directly. It's a lower level building block to some kinds of implementations of fuzzy search.
